### ISSUE TITLE
QTree does not set aria attributes

### ISSUE THREAD

**Issue by Argannor** [quasarframework/quasar#16404]
### What happened?

When using the Tree component with nested nodes screen readers are not advised about the expandability and expand state of the child notes, because corresponding aria attributes are missing, therefore making it more difficult to understand.

I will shortly send a PR addressing this issue.

### What did you expect to happen?

Tree nodes should have the attribute `aria-expanded="true|false"` if they have children.

### Reproduction URL

https://codepen.io/pen?&editors=101&prefill_data_id=ba378ef4-67d3-4f17-ad86-1d3b002bce5a

### How to reproduce?

1. Go to the provided link
2. Open and close the expandable tree items
3. Either use a screen reader or inspect the aria-expanded attributes (or lack therof)

### Flavour

Vite Plugin (@quasar/vite-plugin)

### Areas

Components (quasar), Accessibility [a11y] (quasar)
